readme.md

```diff
@@ -83,9 +83,9 @@ Stops the stream on failed tests.
 
 ##### config
 
-Type: `object`
+Type: `object` | `string`
 
-Passes the config to Jasmine's [loadConfig](http://jasmine.github.io/2.4/node.html#section-Load_configuration_from_a_file_or_from_an_object.) method.
+Passes the config to Jasmine's [loadConfig](http://jasmine.github.io/2.4/node.html#section-Load_configuration_from_a_file_or_from_an_object.) or [loadConfigFile](http://jasmine.github.io/2.4/node.html#section-Load_configuration_from_a_file_or_from_an_object.) method.
```

Comment:
This is incorrect. You're not passing the `config` to `loadConfigFile`, but the path to the config file.